packages/apps-config/src/endpoints/testingRelayRococo.ts

```diff
@@ -393,6 +393,14 @@ export function createRococo (t: TFunction): EndpointOption {
           SubDAONetwork: 'wss://parachain.subdao.network'
         }
       },
+      {
+        info: 'rococoOriginTrail',
+        paraId: 2024,
+        text: t('rpc.origintrail', 'OriginTrail Parachain', { ns: 'apps-config' }),
```

Comment:
Can we just move this up - alphabetical based on chain. (With the number of items it is a pain to try and locate if not done).

Reply:
Good point! Done!